sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -60,6 +60,10 @@ public static IEnumerable<INamedTypeSymbol> GetAllNamedTypes(this INamedTypeSymb
             }
         }
 
+        public static bool IsInherited(this IMethodSymbol symbol) => GetInterfaceMember(symbol) != null;

```

Comment:
I think this is actually quite reusable, but I'm fine in making it private first.